### ISSUE TITLE
Softmax 최고값에대해 맞춘경우 틀린경우 단어에 따라 나눠서 히스토그램 그려보기

### ISSUE THREAD

**Issue by happyhj** [bosspiglet/tensorflow#9]
일단 (예측단어, 실제단어, Softmax  최고값) 순으로 csv 로 저장한 후 
 pandas 같은걸 활용해서 쥬피터 노트북으로 히스토그램 그려보자.

걍 엑셀로 그려도 될듯
